src/test/scala/org/broadinstitute/dsde/firecloud/service/StatusServiceSpec.scala

```diff
@@ -0,0 +1,149 @@
+package org.broadinstitute.dsde.firecloud.service
```

Comment:
because these tests run through the orch api - `Get("/status") ~> sealRoute(publicStatusRoutes) ~> check {`, this file should really be named `StatusApiServiceSpec`, with appropriate class renaming inside